### PR TITLE
chore(logstash-version): Upgrade dependencies to be compatible with Logstash 5.0

### PR DESCRIPTION
**Pull request by mpiercy827**
**What & Why**

Upgrade the plugin to work with Logstash 5.0, because we will be migrating to Logstash 5.0

### CODE REVIEW

File: logstash-codec-bytes.gemspec [hunk 1]
```diff
@@ -15,9 +15,10 @@ Gem::Specification.new do |s|
 
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "codec" }
 
-  s.add_runtime_dependency "logstash-core", ">= 2.0.0", "< 3.0.0"
+  s.add_runtime_dependency "logstash-core-plugin-api", "~> 2.0"
 
   s.add_development_dependency "logstash-devutils"
   s.add_development_dependency "coveralls", "~> 0.8.1"
   s.add_development_dependency "simplecov"
+  s.add_development_dependency "rspec"
```
Comment (mpiercy827) [inline logstash-codec-bytes.gemspec]: Added this line in a naive attempt to fix the build (didn't work) and am looking into a better solution.

File: logstash-codec-bytes.gemspec [hunk 2]
```diff
@@ -15,9 +15,9 @@ Gem::Specification.new do |s|
 
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "codec" }
 
-  s.add_runtime_dependency "logstash-core", ">= 2.0.0", "< 3.0.0"
+  s.add_runtime_dependency "logstash-core-plugin-api", "~> 2.0"
+  s.add_runtime_dependency "logstash-codec-line"
```
Comment (robinjoseph08) [inline logstash-codec-bytes.gemspec]: I wonder if this is actually necessary to include... Does it still work if we don't include it?

File: .travis.yml [hunk 3]
```diff
@@ -1,8 +1,10 @@
+---
```
Comment (mgartner) [inline .travis.yml]: what is this for?

Comment (mpiercy827) [inline .travis.yml]: To be honest, I'm not too sure. Found it in a similar plugin that fixed the same issue we were having with builds, so I added it in. Will see if the build does ok without it.

**Pull request merged**